LayoutTests/TestExpectations

```diff
@@ -2479,9 +2479,6 @@ fast/images/webp-as-image.html [ Skip ]
 fast/images/animated-webp-as-image.html [ Skip ]
 fast/images/animated-webp.html [ Skip ]
 
-# AVIF overlay images are only supported on macOS and iOS post Ventura.
-fast/images/image-overlay-mutiple-frames.html [ Skip ]
```

Comment:
Assuming this comment is correct, this is going to fail on non-Apple ports.

Reply:
Good catch! Fixed in latest commit, thanks.